python/katana/dynamic_bitset.pxd

```diff
@@ -0,0 +1,5 @@
+from .cpp.libgalois.DynamicBitset cimport DynamicBitset as CDynamicBitset
+
```

Comment:
not sure what changed here :)

Reply:
I needed to lift this code into a `.pxd` so I could call it from other Cython code directly.